### ISSUE TITLE
Feature request : Add a what's new like feed for subscriptions.

### ISSUE THREAD

**Issue by bluecheeser** [jonjomckay/fritter#23]
Newpipe has a what's new feed where new videos of subscribers show up. Can something be implemented like that in fritter?

**Comment by backme7** [comment]
That'll be wonderful, a 3rd tab along with "Tending" and "Subscriptions"
The 1st (dafault) tab should be "What's New"

**Comment by jonjomckay** [comment]
Would this functionally be the same as the "All" subscription group or were you thinking of another way? If so, it shouldn't be hard at all to lift that into its own tab 🙂

**Comment by backme7** [comment]
When I open Twitter app, first thing I see is all the new posts from all the profiles. Same goes for all the other social media apps like Barinsta, Infinity, etc.

When someone opens Fritter then first thing he/she should see "ALL" the new posts like every other social media app.

Later this functionality can be added to customize the default tab or any other tab like in Newpipe.

**Comment by bluecheeser** [comment]
> Would this functionally be the same as the "All" subscription group or were you thinking of another way? If so, it shouldn't be hard at all to lift that into its own tab 🙂

Yes exactly this."All" being a separate tab would be way more convenient and will be easier to access subscription feed.
